### PR TITLE
Bus live performance function.

### PR DESCRIPTION
**Pull request by meddle0x53**

### CODE REVIEW

File: app/scripts/transportapi.js [hunk 1]
```diff
@@ -142,6 +142,20 @@
       return transportAPIcall(url, successFunction);
     },
 
+    busPerformance: function(codes, successFunction, timeout) {
+      var codesParam = codes.join(',');
+
+      var url = transportAPIbaseURL +
```
Comment (davidmountain) [inline app/scripts/transportapi.js:148]: Fine for this commit. But looks a  slightly ugly way of constructing a URL with query string parameters? JQuery has methods for this, but we don't want to have to bundle that lib. Any other lib we could reuse?

**Comment by davidmountain** [comment]
We're bundling the dists, because we don't want users of this lib to have to build it themselves right?

File: app/scripts/transportapi.js (hunk 1 above)
Comment (meddle0x53) [inline app/scripts/transportapi.js:148]: You are right, I'll make a re-factoring PR fixing this soon.
The problem is that all the functions have to be changed and the base one too, so it is a re-factoring.

**Comment by davidmountain** [comment]
Overall a merge, but worth thinking about more generic ways of constructing a URL from QS params

**Comment by meddle0x53** [comment]
Yeah, I know ways, we didn't really work on this `lib` yet. I have ideas for it. I think that it can be auto-generated in the future.

**Pull request merged**